frontend/src/__tests__/cypress/cypress/pages/distributedWorkloads.ts

```diff
@@ -29,6 +30,21 @@ class GlobalDistributedWorkloads {
     return cy.findByTestId('project-selector-dropdown');
   }
 
+  findRefreshIntervalSelectToggle() {
+    return cy.get('#metrics-toolbar-refresh-interval-select-toggle');
+  }
+
+  selectRefreshInterval(interval: RefreshIntervalTitle) {
+    this.findRefreshIntervalSelectToggle().click();
+    cy.findByTestId('metrics-toolbar-refresh-interval-select').within(() => {
+      cy.findByText(interval).click();
+    });
+  }
+
+  refreshIntervalShouldBe(interval: RefreshIntervalTitle) {
```

Comment:
Start all assertion functions with `should`.
```suggestion
  shouldHaveRefreshInterval(interval: RefreshIntervalTitle) {
```